e2e-tests/web/wdio-browserstack.conf.ts

```diff
@@ -0,0 +1,18 @@
+import { browsers, browserstackCapabilities } from './capabilities'
+import { config as defaultConfig } from './wdio.conf'
+
+const capabilities = browsers
+  .filter(browser => process.argv.includes(`--${browser}`))
```

Comment:
Shouldn't this rather be a find? Or is it possible that e2e tests are run in multiple browsers?

Reply:
it is possible to run the e2e tests in multiple browsers. check out `yarn test:web --chrome --firefox`